client/src/components/App/UpcomingBookings.tsx

```diff
@@ -51,6 +52,14 @@ const UpcomingBookings: React.FC<RouteComponentProps> = () => {
     }
   }, [bookings]);
 
+  const upcomingBookings = bookings?.filter(
+    (b) => isFuture(parse(b.date, 'y-MM-dd', new Date(), DATE_FNS_OPTIONS)) === true
```

Comment:
Does isFuture return true for today's date? Given we've already gone past 00:00:00 today? Would then then show the current day's bookings in the past bookings?